Indicators/Alpha.cs

```diff
@@ -0,0 +1,257 @@
+        /// </summary>
+        /// <param name="name"></param>
+        /// <param name="targetSymbol"></param>
+        /// <param name="referenceSymbol"></param>
+        /// <param name="period">Period of the indicator - alpha and beta</param>
+        public Alpha(string name, Symbol targetSymbol, Symbol referenceSymbol, int period)
+            : this($"ALPHA({targetSymbol},{referenceSymbol},{period})", targetSymbol, referenceSymbol, period, period)
+        {
+        }
+        
+        /// <summary>
+        /// Computes the next value for this indicator from the given state.
+        /// </summary>
+        /// <param name="input"></param>
+        /// <returns></returns>
+        /// <exception cref="ArgumentException"></exception>
+        protected override decimal ComputeNextValue(IBaseDataBar input)
+        {
+            var inputSymbol = input.Symbol;
+            if (inputSymbol == _targetSymbol)
+            {
+                _targetDataPoints.Add(input.Close);
+            }
+            else if (inputSymbol == _referenceSymbol)
+            {
+                _referenceDataPoints.Add(input.Close);
+            }
+            else
+            {
+                throw new ArgumentException($"The input symbol {inputSymbol} is not the target or reference symbol.");
+            }
+
+            if (_targetDataPoints.Samples == _referenceDataPoints.Samples && _targetDataPoints.IsReady && _referenceDataPoints.IsReady)
+            {
+                _targetAlphaReturns.Add(GetNewReturn(_targetDataPoints));
+                _referenceAlphaReturns.Add(GetNewReturn(_referenceDataPoints));
+
+                _targetBetaReturns.Add(GetNewReturn(_targetDataPoints));
+                _referenceBetaReturns.Add(GetNewReturn(_referenceDataPoints));
+
+                ComputeAlpha();
+            }
+            return _alpha;
+        }
+
+        /// <summary>
+        /// Computes the return of the last two data points of the window
+        /// </summary>
+        /// <param name="returns"></param>
+        /// <returns>Returns the return of the last two data points of the window</returns>
+        /// <exception cref="ArgumentException"></exception>
+        public static double GetNewReturn(RollingWindow<decimal> returns)
+        {
+            // Assert that the window has at least two data points
+            if (returns.Count < 2)
+            {
+                throw new ArgumentException("The window must have at least two data points.");
+            }
+
+            // Return of the last two data points of the window
+            // (last - previous) / previous
+            return (double)((returns[0] - returns[1]) / returns[1]);
+        }
+
+        /// <summary>
+        /// Computes the alpha of the target used in relation with the reference and stores it in the _alpha field
+        /// </summary>
+        private void ComputeAlpha()
```

Comment:
See the comments in the original issue, this Alpha indicator can be simplified by using our existing `Beta` and `RateOfChange` indicators.

Reply:
Thanks for the suggestion. I chose not to reuse the existing code initially because I thought my approach was a better fit for the specific requirements of the Alpha indicator. However, I understand the benefits of using the established Beta and RateOfChange indicators for consistency and efficiency. I'll revise my implementation to incorporate these.